src/test/groovy/org/transmartproject/batch/highdim/proteomics/platform/ProteomicsAnnotationRowValidatorTests.groovy

```diff
@@ -35,8 +36,9 @@ class ProteomicsAnnotationRowValidatorTests {
 
     @Before
     void setUp() {
-        platform
-        testee = new ProteomicsAnnotationRowValidator(platformObject: platform)
+        testee = new ProteomicsAnnotationRowValidator()
+        testee.org_transmartproject_batch_highdim_datastd_ChromosomalRegionValidator__platformObject = platform
+        testee.org_transmartproject_batch_highdim_datastd_PlatformOrganismValidator__platformObject = platform
```

Comment:
What's wrong with the previous snippet. According to the docs, conflicts are resolved in Groovy traits by picking the last included class.